ios/CovidShield/ExposureNotification.m

```diff
@@ -115,6 +115,25 @@ - (void)invalidate
   }];
 }
 
+RCT_REMAP_METHOD(getExposureWindows, getExposureWindowsFromSummary: (ENExposureDetectionSummary *)summary resolver:(RCTPromiseResolveBlock)resolve rejecter:(RCTPromiseRejectBlock)reject) {
```

Comment:
Can we call this `getExposureWindowsFromSummary` rather than `getExposureWindows`? I was running into problems with the type definition calling from react native, since the android fn by the same name does not take an argument.